packages/venia-ui/lib/components/CartPage/ProductListing/product.js

```diff
@@ -77,12 +79,14 @@ const Product = props => {
                     isFilled={isFavorite}
                     classes={{ text: classes.sectionText }}
                 />
-                <Section
-                    text="Edit item"
-                    onClick={handleEditItem}
-                    icon="Edit2"
-                    classes={{ text: classes.sectionText }}
-                />
+                {isEditable ? (
```

Comment:
Checkout #2200 for conditional rendering. You can move this out of return, put it in a `useMemo` and place the returned out of `useMemo` here.

Reply:
Good to know, fixed in fbe5519.